about.html

```diff
@@ -12,62 +13,114 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="nav_bar">
+    <img src="./img/lambda-black.png" alt="Lambda School" />
+    <nav>
```

Comment:
A more semantic way of introducing navigation links or introductory content is by using the  <header> element tag. Nonetheless, good job organizing the Navbar